handbook/communication/seeking-and-giving-feedback.md

```diff
@@ -0,0 +1,50 @@
+# Seeking and giving feedback
+
+Seeking and acting on feedback is crucial for making good decisions and delivering our best work. Good feedback is a product of an intentional environment with positive feedback loops.
+
+## Seeking feedback
+
+When seeking feedback at Sourcegraph, it's important to:
+
+- Specifically say what *kind* of feedback you're seeking. A thumbs-up, thumbs-down? A gut call on the better of two approaches? A detailed critique?
+- Ask *specific questions* that you want feedback about.
+- Provide *context*. Make sure the specific problem and goal is clear.
+- Seek feedback according to [project roles](#project-roles).
+- After receiving feedback, follow up. Recognize contributions and share how you will move forward.
+
+### Great examples of requesting feedback
+
+(To be added!)
+
+## Giving feedback
+
+When giving feedback, it's important to:
+
+- Make sure you understand what kind of feedback is sought. If it's not clear what kind of feedback is sought, clarify before giving feedback.
+- Focus feedback on the specific feedback sought. Other feedback may be valuable, but should be separated into a distinct conversation.
+- Provide feedback in terms of **why**, not **how** (unless specifically sought).
```

Comment:
I agree with this line in general but I'd like to make it clear here that it's ok to suggest a potential solution @quinnkeast. Maybe we can describe it:

'When providing feedback always start with **why**. You're welcomed to suggest a potential solution (the **how**) but you need to describe your reasoning first so that your proposition is easily understandable.'

Reply:
Maybe something like:

```suggestion
- Always start with **why**, not **how**. Potential solutions can be useful to clarify your feedback, but aren't feedback in themselves.
```

Reply:
Looks good, thank you @quinnkeast !